integration-tests/oidc-wiremock/src/test/resources/privateKey.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
@cemnura use JWK format here, copy this file,
https://github.com/smallrye/smallrye-jwt/blob/master/testsuite/basic/src/test/resources/decryptPrivateKey.jwk (just rename ir to `privateKey.jwk`)